src/main/java/at/ac/tuwien/kr/alpha/grounder/structure/AnalyzeUnjustified.java

```diff
@@ -89,8 +110,7 @@ private ReturnExplainUnjust explainUnjust(LitSet x, Assignment currentAssignment
 		// Construct set of all 'rules' such that head unifies with p.
 		List<RuleAndUnifier> rulesUnifyingWithP = rulesHeadUnifyingWith(p);
 		log("Rules unifying with {} are {}", p, rulesUnifyingWithP);
-		rulesLoop:
-		for (RuleAndUnifier ruleUnifier : rulesUnifyingWithP) {
+		rulesLoop: for (RuleAndUnifier ruleUnifier : rulesUnifyingWithP) {
```

Comment:
Please revert the formatting.

Reply:
done

Reply:
There appears now to be a space behind the label and the `for` is indented (which I think it should not be).

Reply:
should be fixed now